addons/pos_restaurant/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -14,10 +14,16 @@
                         <span class="d-block">Order</span>
                         <small><t t-esc="nbrOfChanges"/> changes</small>
                     </button>
+                    <button
+                            class="h-100 button btn btn-light btn-lg d-flex flex-row align-items-center justify-content-center"
+                            t-on-click="() => this.submitOrder()"
+                            t-elif="this.pos.unwatched.printers.length and this.currentOrder.uiState.lastPrint">
+                        <i class="fa fa-print" aria-hidden="true"></i>
```

Comment:
this icon is the same as early printing receipt. You could maybe change one of them to not confuse the user